javascript/organizing_your_javascript_code/objects_and_object_constructors.md

```diff
@@ -36,6 +36,8 @@ myObject[variable]; // this is equivalent to myObject['property'] and returns 'V
 
 If you are feeling rusty on using objects, now might be a good time to go back and review the content in our [object basics lesson](https://www.theodinproject.com/lessons/foundations-object-basics) from our JavaScript Basics course.
 
+You should also read JavaScript.Info's article [Object references and copying](https://javascript.info/object-copy) to learn more about how objects in JavaScript behave differently than primitive values you may be used to using up to this point in the curriculum.
```

Comment:
We shouldn't really be linking to resources like this in an introduction of a lesson. Due to the context of this lesson this topic may not really be ideal here, but rather the object basics lesson linked on line 37. Though the resource you link goes into a bit more than we may want users to dive into (the content on object assign and structuredClone).

Reply:
An option might be to add a section to the beginning of the lesson, something like:
```markdown
### Object references

You may or may not have noticed that objects and object-like data types in JS behave differently than the primitive values you may be used to. Read up about object references in JavaScript.Info's [Object references and copying](https://javascript.info/object-copy) article 
```

Or, I could make a PR to add this resource for the lesson in Foundations, if you don't feel it's too early to explain and have users exposed to `Object.assign` and `structuredClone` in the context of this article.

Reply:
I still think the lesson in Foundations would be a better spot for the concept of references. If there's a resource that covers it without going into stuff like assign and structuredClone that'd be even better, but if we explicitly tell users to only read X/not to read Y then it isn't the worst thing.